superset-frontend/src/dashboard/components/RefreshIntervalModal.tsx

```diff
@@ -111,17 +165,79 @@ class RefreshIntervalModal extends React.PureComponent<
         modalTitle={t('Refresh interval')}
         modalBody={
           <div>
-            <FormLabel>{t('Refresh frequency')}</FormLabel>
-            <Select
-              ariaLabel={t('Refresh interval')}
-              options={refreshIntervalOptions.map(option => ({
-                value: option[0],
-                label: t(option[1]),
-              }))}
-              value={refreshFrequency}
-              onChange={this.handleFrequencyChange}
-              sortComparator={propertyComparator('value')}
-            />
+            <div id="refresh_from_dropdown">
+              <FormLabel>
+                <b>{t('Refresh frequency')}</b>
+              </FormLabel>
+              <Select
+                ariaLabel={t('Refresh interval')}
+                options={this.createIntervalOptions(refreshIntervalOptions)}
+                value={refreshFrequency}
+                onChange={this.handleFrequencyChange}
+                sortComparator={propertyComparator('value')}
+              />
+            </div>
+            {custom_block && (
+              <div
+                style={{
+                  display: 'flex',
+                  marginTop: '15px',
+                }}
```

Comment:
We prefer not to use inline styles, or hard-coded pixel values. We prefer using Emotion (kind of like Styled Components) and using the Theme's gridUnits. There are some style guides available [here](https://github.com/apache/superset/wiki/Emotion-Styling-Guidelines-and-Best-Practices).

Reply:
Done